src/malli/util.cljc

```diff
@@ -399,7 +399,7 @@
      (clojure.core/update children 0 #(m/form % options))
      (apply f (conj children options))]))
 
-(defn -util-schema [{:keys [type min max childs type-properties fn]}]
+(defn -util-schema [{:keys [type min max childs type-properties func]}]
```

Comment:
why the rename?

Reply:
It was conflicting with clojure.core/fn macro I used in first iteration, and because util-schema is private I decided to rename argument rather than using fully qualified name of the macro. Anyway, I reverted this change in the second iteration.